### PR TITLE
Resolve issue #2478 by adding -TimeOut to Test-Connection

### PR DESCRIPTION
**Pull request by jpsnover**

### CODE REVIEW

File: src/Microsoft.PowerShell.Commands.Management/commands/management/Computer.cs [hunk 1]
```diff
@@ -128,6 +128,14 @@ public class TestConnectionCommand : PSCmdlet
         public Int32 BufferSize { get; set; } = 32;
 
         /// <summary>
+        /// The following is the definition of the input parameter "BufferSize".
+        /// Buffer size sent with the this command. The default value is 32.
```
Comment (mirichmo) [inline src/Microsoft.PowerShell.Commands.Management/commands/management/Computer.cs]: This comment must be updated. It was copied and pasted without modification.

**Review by mirichmo — CHANGES_REQUESTED** [review]

**Comment by mirichmo** [comment]
@jpsnover Please include test(s) for this change in ...\test\powershell\Modules\Microsoft.PowerShell.Management

@adityapatwardhan and @JamesWTruher Please review

**Comment by daxian-dbw** [comment]
@jpsnover PRs should be sent out from a fork instead of from a branch directly pushed to powershell/powershell. That is to make sure powershell/powershell has a minimal number of branches. Could you please close this PR and send it from a fork after addressing the CR comments?

File: src/Microsoft.PowerShell.Commands.Management/commands/management/Computer.cs [hunk 2]
```diff
@@ -128,6 +128,14 @@ public class TestConnectionCommand : PSCmdlet
...
+        /// </summary>
+        [Parameter]
+        [ValidateRange((int)1, Int32.MaxValue)]
+        public Int32 TimeOut { get; set; } = 1000;
```
Comment (JamesWTruher) [inline src/Microsoft.PowerShell.Commands.Management/commands/management/Computer.cs]: >1000 [](start = 45, length = 4)

I'm a little worried about this - how did you land on this value? Could the default value be Int32.MaxValue to maintain current behavior? Anyone using this new parameter would likely provide a much smaller number in any case

**Comment by jpsnover** [comment]
This was superseded by PR #2492

**Pull request closed without merging**